README.md

```diff
@@ -48,7 +48,7 @@ All docker-compose env vars are able to be set in both `make start-trade-bot` an
     - `GAS_ADJUSTMENT`: how much more than the base estimated gas price to pay for each tx
     - `GAS_PRICES`: calculate how many fees to pay from this fraction of gas
     - `TOKEN_CONFIG`: a token pairs configuration (JSON) object for eg. token amounts to trade
-        - see [helpers.sh](https://github.com/neutron-org/dex-trading-bot/blob/e0f6f7128182b9dce2a54abbee279219ae8dc9fc/scripts/helpers.sh#L41-L59) for more setting details
+        - see [helpers.sh](https://github.com/neutron-org/dex-trading-bot/blob/e0f6f7128182b9dce2a54abbee279219ae8dc9fc/scripts/helpers.sh#L41-L63) for more setting details
```

Comment:
oh, that was bad of me. for now we can do that, but this is another sign that having part of the app documentation in code, not in readme, is wrong. imagine we committed this suggestion of yours, and then pulled your order-agnostic PR which changes the description a bit (renames token0/1 to tokenA/B), and we'll have to edit this line, too, because the commit hash would be outdated again

Reply:
Yeah, you're right. The section should be moved. 

I can move it in a different PR that improves the docs with the other usage notes I've made so far in https://github.com/neutron-org/dex-trading-bot/compare/c0fd70041051f4e3d1cdea655160d3f78d8ab1d1~..5464dff6cf60e60ba4368185cd463b702e3eacf3